content/v2.0/reference/flux/stdlib/pushbullet/pushdata.md

```diff
@@ -0,0 +1,79 @@
+---
+title: pushbullet.pushData() function
+description: >
+  The `pushbullet.pushData()` function sends a push notification to the Pushbullet API.
+menu:
+  v2_0_ref:
+    name: pushbullet.pushData
+    parent: Pushbullet
+weight: 202
+---
+
+The `pushbullet.pushData()` function sends a push notification to the Pushbullet API.
+
+_**Function type:** Output_
+
+```js
+import "pushbullet"
+
+pushbullet.pushData(
+  url: "https://api.pushbullet.com/v2/pushes",
+  token: "",
+  data: {
+    "type": "link",
```

Comment:
Does Pushbullet support data types other than link and note?

Reply:
yep, I think it does: https://docs.pushbullet.com/#create-push

Reply:
Yes, it does. This is just an example of data being sent to the Pushbullet API. All the different options are documented in their API documentation. I'll add a link.